client/src/components/common/assets/arrowUp.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Again - try to avoid similar images with different directions - rotate existing ones.
Same goes for colors - use fill